scripts/templates/breaking-change.ejs

```diff
@@ -0,0 +1,17 @@
+<li>
+<%= content %>
+<%
+      for (const reference of commit.references) {
```

Comment:
ejs supports js build in functions like forEach so we can use it:

```
commit.references
  .filter(reference => !reference.action || !reference.issue)
  .forEach(reference => {
    // ...
  });
```
PS. I don't know are arrow functions allowed here

Reply:
Tried and this doesn't work. Using regular function also ended up with template compilation error.